internal/internal_worker_test.go

```diff
@@ -425,6 +433,13 @@ func createWorkerWithAutoscaler(
 	return createWorkerWithThrottle(t, service, float64(0), WorkerOptions{FeatureFlags: FeatureFlags{PollerAutoScalerEnabled: true}})
```

Comment:
```suggestion
	return createWorkerWithThrottle(t, service, 0, WorkerOptions{FeatureFlags: FeatureFlags{PollerAutoScalerEnabled: true}})
```
maybe this too while you're here